saleor/graphql/webhook/mutations.py

```diff
@@ -4,13 +4,29 @@
 from ...core.permissions import AppPermission, AuthorizationFilters
 from ...webhook import models
 from ...webhook.error_codes import WebhookErrorCode
-from ..core.descriptions import DEPRECATED_IN_3X_INPUT
+from ...webhook.subscription_payload import validate_subscription_query
+from ..core.descriptions import DEPRECATED_IN_3X_INPUT, PREVIEW_FEATURE
 from ..core.mutations import BaseMutation, ModelDeleteMutation, ModelMutation
 from ..core.types import NonNullList, WebhookError
 from . import enums
 from .types import EventDelivery, Webhook
 
 
+def validate_query(query):
+    if not query:
+        return
+    is_valid = validate_subscription_query(query)
+    if not is_valid:
+        raise ValidationError(
+            {
+                "query": ValidationError(
```

Comment:
The one thing which I am worried about is that we don't provide a lot of info. We just say that something was wrong and you need to figure it out. We could somehow wrap a msg from graphql validator (something similar that we get when we will try to send an incorrect query).
Maybe it is not the scope of this PR but it is worth mentioning it.
@maarcingebala  what do you think?

Reply:
Definitely would be useful to give more info but don't have to be done in this PR.

Reply:
As we agreed - this is something to be further discussed and done in future PRs.